src/components/community/BrandFeed.jsx

```
@@ -0,0 +1,152 @@
+            console.debug?.('BrandFeed enrich failed', err);
+            return { ...post };
+          }
+        }));
+
+          setPosts(enriched);
+          // Surface filters upward so the left rail can reflect current brand/tags
+          try {
+            const brands = brandName ? [brandName] : [];
+            const tags = Array.from(new Set(enriched.flatMap(p => Array.isArray(p.tags) ? p.tags : []).filter(Boolean)));
+            onFiltersChange?.({ brands, tags, tagCounts: {} });
+          } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)